static/js/components/forms/ChangeEmailForm.js

```diff
@@ -47,19 +47,27 @@ const ChangeEmailForm = ({ onSubmit, user }: Props) => (
             <label htmlFor="email">Email</label>
             <Field
               name="email"
+              id="email"
               className="form-control"
               component={EmailInput}
+              aria-describedby="emailError"
             />
-            <ErrorMessage name="email" component={FormError} />
+            <ErrorMessage name="email" id="emailError" component={FormError} />
           </div>
           <div className="form-group">
             <label htmlFor="confirmPassword">Confirm Password</label>
             <Field
+              id="confirmPassword"
               name="confirmPassword"
               className="form-control"
               component={PasswordInput}
+              aria-describedby="confirmPasswordError"
+            />
+            <ErrorMessage
+              name="confirmPassword"
```

Comment:
(According to my knowledge)
The name attribute is not passed  to our custom component and is used by Formik internally for associating the fields name in Formik state.
The name attribute is also required in case for ErrorMessage component.
Reference: https://formik.org/docs/api/errormessage